lib/generators/awesome_rails_console/install/install_generator.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails/generators/base'
+
+class AwesomeRailsConsole::InstallGenerator < Rails::Generators::Base
+  source_root File.expand_path('../templates', __FILE__)
+
+  def update_gemfile
+    gem_group :development, :test do
+      gem 'hirb'
+      gem 'hirb-unicode'
+      gem 'pry-byebug'
+      gem 'pry-stack_explorer'
+    end
+
+    inject_into_file 'Gemfile', comment, before: "group :development, :test do\n"\
+                                                 "  gem 'hirb', require: false\n"
+  end
+
+  def write_to_pryrc_file
+    copy_file '.pryrc', '.pryrc'
+  end
+
+  private
+
+  def comment
+     "# Please clean up duplicated gems if any. "\
+       "Feel free to remove gems that you don't want to use or "\
```

Comment:
Perhaps this: Feel free to remove gems that you don't want to use or if they conflict with other gem dependencies.